src/cdk-experimental/menu/menu-stack.ts

```diff
@@ -86,4 +106,19 @@ export class MenuStack {
       this._empty.next(focusNext);
     }
   }
+
+  /** Return true if this stack is empty. */
+  isEmpty() {
+    return this._elements.length === 0;
```

Comment:
optionally could do:
```suggestion
    return !!this._elements.length;
```